.github/workflows/consistency-functional.yml

```diff
@@ -268,6 +268,11 @@ jobs:
               --src devstack \
               --dst devstack-alt \
               > tests/auth_tenant.yml && \
```

Comment:
On each PR we will run both admin and tenant, maybe we can split the job in two with a dependence any way this looks like a nice improvement

Reply:
+1